tendermint/src/signature.rs

```diff
@@ -27,6 +30,9 @@ impl TryFrom<Vec<u8>> for Signature {
     type Error = Error;
 
     fn try_from(value: Vec<u8>) -> Result<Self, Self::Error> {
+        if value.is_empty() {
```

Comment:
The value sent from Go Tendermint over privval socket (for signing) in Vote/Proposal requests is actually not empty: https://github.com/informalsystems/tendermint-rs/issues/626#issue-717983039 -- it has one element which is `0u8`.
tmkms could do some preprocessing though and clear that value before creating a domain type

Reply:
Apologies, I misunderstood your comment in the issue. And, come on, a single `0u8`? Aaaargh.

I'll add that in the code. One more thing to raise to the Go team. Thanks!

Reply:
oops, my apologies -- the `vec![0u8]` is something I saw in a debugger when running the Go unit tests. I've just tried against the TM process (0.34.0-rc5) and it's indeed empty.
735c153faf19133305f8ab3adad9385220999a4d is OK
(9ed45f012a1d2d76b27d9d49858217f60d8281f9 can be reverted)

Reply:
LOL, thanks for checking again! I'm happy to implement extra work if it leads to extra checks! I should have checked it myself...